Avoid downloading of files from inet at build time
This is provided by hwdata packageIndex: lshw-02.14/src/Makefile
===================================================================
Index: lshw/src/Makefile
===================================================================
--- lshw.orig/src/Makefile
+++ lshw/src/Makefile
@@ -97,16 +97,16 @@ $(PACKAGENAME).1: $(PACKAGENAME).sgml
 	docbook2man $<
 
 pci.ids:
-	wget http://pciids.sourceforge.net/pci.ids
+	#wget http://pciids.sourceforge.net/pci.ids
 
 usb.ids:
-	wget http://www.linux-usb.org/usb.ids
+	#wget http://www.linux-usb.org/usb.ids
 
 oui.txt:
-	wget http://standards.ieee.org/regauth/oui/oui.txt
+	#wget http://standards.ieee.org/regauth/oui/oui.txt
 
 manuf.txt:
-	wget -O $@ http://anonsvn.wireshark.org/wireshark/trunk/manuf
+	#wget -O $@ http://anonsvn.wireshark.org/wireshark/trunk/manuf
 
 pnp.ids:
 	wget https://git.fedorahosted.org/cgit/hwdata.git/plain/pnp.ids
